### PR TITLE
Unit Test Addition for 503_VerifyMinimumSchema

### PR DESCRIPTION
**Pull request by souravbaner-da**

### CODE REVIEW

File: src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala [hunk 1]
```diff
@@ -433,7 +433,7 @@ object SchemaTools extends SparkSessionWrapper {
                       validator: ValidatedColumn,
                       cPrefix: Option[String] = None,
                       enforceNonNullCols: Boolean = true,
-                      isDebug: Boolean = false
+                      isDebug: Boolean = false,
```
Comment (GeekSheikh) [inline src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala:436]: remove this comma.

File: src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala [hunk 2]
```diff
@@ -496,9 +496,32 @@ object SchemaTools extends SparkSessionWrapper {
                 isDebug,
                 newPrefix
               ).map(validateSchema(_, newPrefix))
+//              println("1st Item",validatedChildren(0))
+//              println("validatedChildren1 is ",struct(validatedChildren1.map(_.column): _*).alias(fieldStructure.name))
+//              println("fieldStructure.name is ",fieldStructure.name)
+//              println("dtArray.elementType is ",dtArray.elementType)
+
+//              println("validatedChildren is ",struct(validatedChildren.map(_.column): _*).alias(fieldStructure.name))
```
Comment (GeekSheikh) [inline src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala:504]: cleanup all these old commented out lines.

Comment (GeekSheikh) [reply src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala:504]: same for whole file.

**Review by GeekSheikh — CHANGES_REQUESTED** [review]
looks good -- just a few items. Please also send me the spreadsheet for these test cases.

Thanks.

File: src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala (hunk 1 above)
Comment (souravbaner-da) [reply src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala:436]: @GeekSheikh  Done

File: src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala (hunk 2 above)
Comment (souravbaner-da) [reply src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala:504]: Done

**Comment by souravbaner-da** [comment]
> 

@GeekSheikh Has sent you the spreadsheet for the testcases.

**Pull request closed without merging**